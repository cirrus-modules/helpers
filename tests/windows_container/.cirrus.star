load("../../lib.star", "task", "windows_container")

def main(ctx):
    return [
        task("task default windows container", windows_container()),
        task("task custom windows container",
             windows_container("cirrusci/windowsservercore:visualstudio2019")),
        task("task custom windows container custom version",
             windows_container("python:3.8-windowsservercore", "2019")),
    ]
