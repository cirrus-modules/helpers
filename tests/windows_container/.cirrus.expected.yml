task:
  name: task default windows container
  windows_container:
    image: cirrusci/windowsservercore
task:
  name: task custom windows container
  windows_container:
    image: cirrusci/windowsservercore:visualstudio2019
task:
  name: task custom windows container custom version
  windows_container:
    image: python:3.8-windowsservercore
    os_version: "2019"
  debug_script:
    - ps: Get-Location
task:
  name: task with unnamed powershell script
  windows_container:
    image: python:3.8-windowsservercore
    os_version: "2019"
  script:
    - ps: Get-Location
